docs/modules/ROOT/pages/api/erc721.adoc

```diff
@@ -208,34 +208,34 @@ NOTE: Implementing xref:api/introspection.adoc#SRC5Component[SRC5Component] is a
 --
 [.sub-index#ERC721Component-Embeddable-Impls-ERC721Impl]
 .ERC721Impl
-* xref:#IERC721-balance_of[`++balance_of(self, account)++`]
-* xref:#IERC721-owner_of[`++owner_of(self, token_id)++`]
-* xref:#IERC721-safe_transfer_from[`++safe_transfer_from(self, from, to, token_id, data)++`]
-* xref:#IERC721-transfer_from[`++transfer_from(self, from, to, token_id)++`]
-* xref:#IERC721-approve[`++approve(self, to, token_id)++`]
-* xref:#IERC721-set_approval_for_all[`++set_approval_for_all(self, operator, approved)++`]
-* xref:#IERC721-get_approved[`++get_approved(self, token_id)++`]
-* xref:#IERC721-is_approved_for_all[`++is_approved_for_all(self, owner, operator)++`]
+* xref:#ERC721Component-balance_of[`++balance_of(self, account)++`]
+* xref:#ERC721Component-owner_of[`++owner_of(self, token_id)++`]
+* xref:#ERC721Component-safe_transfer_from[`++safe_transfer_from(self, from, to, token_id, data)++`]
+* xref:#ERC721Component-transfer_from[`++transfer_from(self, from, to, token_id)++`]
+* xref:#ERC721Component-approve[`++approve(self, to, token_id)++`]
+* xref:#ERC721Component-set_approval_for_all[`++set_approval_for_all(self, operator, approved)++`]
+* xref:#ERC721Component-get_approved[`++get_approved(self, token_id)++`]
+* xref:#ERC721Component-is_approved_for_all[`++is_approved_for_all(self, owner, operator)++`]
 
 [.sub-index#ERC721Component-Embeddable-Impls-ERC721MetadataImpl]
 .ERC721MetadataImpl
-* xref:#IERC721Metadata-name[`++name(self)++`]
-* xref:#IERC721Metadata-symbol[`++symbol(self)++`]
-* xref:#IERC721Metadata-token_uri[`++token_uri(self, token_id)++`]
+* xref:#ERC721Component-name[`++name(self)++`]
+* xref:#ERC721Component-symbol[`++symbol(self)++`]
+* xref:#ERC721Component-token_uri[`++token_uri(self, token_id)++`]
```

Comment:
isn't this adding a level of indirection?

Reply:
`ERC721Component-token_uri` has additional info regarding its implementation (with the URI concatenation strategy that is not included in the `IERC721Metadata-token_uri` description). With `name` and `symbol` though, you are correct